pom.xml

```diff
@@ -365,6 +365,26 @@
         <configuration>
           <showLogs>true</showLogs>
           <images>
+            <image>
+              <name>joeyjackson/kafka:${project.version}</name>
```

Comment:
Probably shouldn't be `joeyjackson`. I assume we need to do this to avoid a name collision with the one in Dockerhub? 

In that case I would say let's use `arpnetworking/mad/kafka`. 

https://hub.docker.com/r/arpnetworking/mad

However, you need to disable `push` for this image. I believe you can do this by using the `filter` condition in the push execution block above.

https://dmp.fabric8.io/#global-configuration

e.g.
```
          <execution>
            <id>default-push</id>
            <phase>deploy</phase>
            <goals>
              <goal>push</goal>
            </goals>
          </execution>
```

You can add a `<configuration>` block to that which will be merged with the one below. The trick is you need to set the merge policy on `<images>` to not append the execution specific image, but to merge it.

You can find more information about Maven's property management here:

https://blog.sonatype.com/2011/01/maven-how-to-merging-plugin-configuration-in-complex-projects/